README.md

```diff
@@ -67,17 +78,18 @@ resource "aws_ami_from_instance" "example" {
 | `security_groups`            | []             | List of Security Group IDs allowed to connect to creating instance        | Yes |
 | `subnets`                    | []             | List of VPC Subnet IDs creating instance launched in                      | Yes |
 | `zone_id`                    | ``             | ID of the domain zone to use - is a result of tf_domain output            | Yes |
+| `associate_public_ip_address`| `true`         | Define if the `public_ip` will be created for the instance                | No  |
```

Comment:
1. Navigate to [tf_instance](https://github.com/cloudposse/tf_instance/blob/master/README.md)
2. Copy description of `associate_public_ip_address` in  [tf_instance](https://github.com/cloudposse/tf_instance/blob/master/README.md)
3. Paste it into `tf_admin` `associate_public_ip_address` description.